ios/Peepz/Peepz.xcodeproj/project.pbxproj

```diff
@@ -816,6 +812,14 @@
 				kind = branch;
 			};
 		};
+		B866911B255D8B0100CABD4A /* XCRemoteSwiftPackageReference "SDWebImageSwiftUI" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/SDWebImage/SDWebImageSwiftUI.git";
+			requirement = {
+				kind = upToNextMajorVersion;
+				minimumVersion = 1.5.0;
+			};
+		};
```

Comment:
That's good to know. I was wondering were Package file went. Obv they packed it to `.pbxproj` because this files is still not complex enough :D 
I think we should refactor and use `Package` instead :)